Example/Analytics/DemoAppAnalytics.swift

```diff
@@ -28,7 +29,7 @@ internal enum DemoAppAnalyticsParameter: String, CaseIterable {
 public class DemoAppAnalytics {
     public static let notificationName = Notification.Name("com.rakuten.esd.sdk.events.custom")
     open class var sdkVersion: String? {
-        Bundle.miniAppSDKBundle.infoDictionary?["CFBundleShortVersionString"] as? String
+        Bundle.miniAppSDKBundle.infoDictionary?["CFBundleShortVersionString"] as? String ?? MiniApp.version
```

Comment:
`CFBundleShortVersionString` is not generated in the plist by SPM. I automated a `MiniApp.version` variable that will be changed every time we run `fastlane create_release` to release the SDK or when we run `./scripts/build-framework.sh` to build a xcframework